README.md

```diff
@@ -16,14 +16,18 @@ npm install
 After completing above installation instructions, to run locally without Basic Authentication:
 
 ``` 
-DEBUG=rubbernecker:* PIVOTAL_API_KEY=your-api-key PIVOTAL_PROJECT_ID=your-project-id npm start
+DEBUG=rubbernecker:* PIVOTAL_API_KEY=your-api-key PIVOTAL_PROJECT_ID=your-project-id PAGERDUTY_API_TOKEN=your-pagerduty-token npm start
 ```
 
 If you want to try out the stunning Basic Auth (useful for when you don't want everyone getting straight to your story overviews)
 ``` 
-DEBUG=rubbernecker:* PIVOTAL_API_KEY=your-api-key PIVOTAL_PROJECT_ID=your-project-id USE_AUTH=true USERNAME=username PASSWORD=password npm start
+DEBUG=rubbernecker:* PIVOTAL_API_KEY=your-api-key PIVOTAL_PROJECT_ID=your-project-id PAGERDUTY_API_TOKEN=your-pagerduty-token USE_AUTH=true USERNAME=username PASSWORD=password npm start
 ```
 
+#### PagerDury Support
+
+Rubbernecker supports PagerDuty, for pulling out the current rota information. This requires a `PAGERDUTY_API_TOKEN` environment variable, to be passed before the app is run.
```

Comment:
Does it fail gracefully if not provided? Can we say that it's optional for people that don't want to use it?